pkg/application/service/services.go

```diff
@@ -11,7 +11,7 @@ type SignupService interface {
 	GetSignup(userID string) (*signup.Signup, error)
 	GetUserSignup(userID string) (*v1alpha1.UserSignup, error)
 	UpdateUserSignup(userSignup *v1alpha1.UserSignup) (*v1alpha1.UserSignup, error)
-	PhoneNumberAlreadyInUse(userID, e164PhoneNumber, email string) error
+	PhoneNumberAlreadyInUse(userID, value string) error
```

Comment:
`value` can be both a hashed phone number or a plain phone number string, right? Can we name the param accordingly to avoid confusion? Or at least add a comment?

Reply:
Correct.  Do you have any suggestions for a better param name? I can certainly add a comment.

Reply:
`phoneNumberOrHash string`? And some comments in the implementation function would be helpful too.

Reply:
I've improved the naming and added more descriptive comments.